### ISSUE TITLE
Display image dimensions (width and height) on profile page image place holder on mouse over

### ISSUE THREAD

**Issue by oliverzet** [Human-Connection/Nitro-Web#65]
At the moment it is unclear which dimensions the large image to be uploaded may have on the profile page. 

The dimensions (width and height) of the image to be uploaded should be displayed in the placeholder on mouse over.

Alternatively it might be preferable for the user to chose the image segment, to be displayed, himself after the upload.
![hc feature request profile image](https://user-images.githubusercontent.com/45514309/49332844-c900ee80-f5b4-11e8-9797-d752de16ee25.png)

**Comment by roschaefer** [comment]
@oliverzet this is an amazing feature and you even added a screenshot with a mocked UI. Adding screenshots with a mock UI increases the attractiveness for someone in the open source community to take a stab on it.

After you added this feature request, you continued to add #67. See? It's good to sign up at Github and become an active part in the development even as a non-technical user. :+1: [Free and open source software](https://en.wikipedia.org/wiki/Free_software) is different from proprietary software as it is controlled and driven by the community and not by a private company whose interests may conflict with the interests of the users.

So thank you for this feature request! I searched on the internet for "writing good feature requests on Github" and I tend to like this tutorial here: https://github.com/rstudio/rstudio/wiki/Writing-Good-Feature-Requests so I suggest you to read through it and continue to submit amazing feature requests. :+1:

**Comment by roschaefer** [comment]
@oliverzet regarding this feature request: The problem is that you want to know in advance what is the ratio of the target image? To avoid a random cropping done by the system? You can now go ahead and improve your feature request by clicking on the "Edit" icon in the top right corner.

**Comment by oliverzet** [comment]
> @oliverzet regarding this feature request: The problem is that you want to know in advance what is the ratio of the target image? To avoid a random cropping done by the system?

Thanks for rephrasing. Yes that is exactly what I want. I've edited the issue and I hope it's a bit more fitting now.

**Comment by appinteractive** [comment]
To be honest, I don't think that showing dimensions will help many people. Even more as the image is responsive and do not have static dimensions like on the big blue network. 

A way better possibility would be to have the option to crop the image before upload (in an overlay). No one wants to crop its images manually in some random tool. 
There are even posibilities to set the point of interesst so that It would always stay in focus, but thats another thing.

So my suggestion is a cropping dialoge instead of dimensions. From that also non technical users would benefit and believe me, thats the biggest group of users, so visually is always better.

**Comment by oliverzet** [comment]
Yes, that's what I suggested as an alternative:

> Alternatively it might be preferable for the user to display a preview of the cropped target image, and make it adjustable, so the user can fit it to its liking.

**Comment by roschaefer** [comment]
However a hint with the ratio would be good enough as a start. The dialog with an editor to crop the image is much more development work!

**Comment by oliverzet** [comment]
Well, I put the issue here, because I was told to do so. If this issue is better suited for "Human-Connection" or "WebApp", please move it to one of these destinations (if possible)

**Comment by roschaefer** [comment]
Yes, we're not talking about prioritization. But writing amazing user-stories is hard work, as you know :wink: so I'm quite happy to have this user-story in the queue and thankful for @oliverzet to document it. :+1:

**Comment by appinteractive** [comment]
Yeah I'm also thankfull for the input! 🤟 Just trying to discuss the best possible way 😃
